src/Components/Carousel.js

```diff
@@ -3,14 +3,20 @@ import GameCard from './GameCard';
 import { uid } from 'react-uid';
 
 class Carousel extends Component {
+
+  shiftCarousel(event) {
+    const shiftDirection = event.target.classList.contains('fa-angle-right') ? 1 : -1;
+    const currentPosition = event.target.parentElement.childNodes[2].scrollLeft;
```

Comment:
Grabbing the parent carousel from whichever button is clicked

Reply:
Nice use of a ternary here!